docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,305 @@
+
+* A laptop or desktop that will be running the dcrd node, referred to as `Computer A`.
+* A secure system that will be running the wallet, referred as `Wallet-B`.
+  This guide will use a Raspberry Pi 4, however it could also be a laptop or desktop.
+  This system should be connected to an external monitor and keyboard, and it
+  should also connect to the local network via ethernet.
+* All details/commands that come within curly braces `{}` cannot be copy-pasted,
+  you will need to remove the curly braces and edit them to suit your setup.
+
+### Benefits of this Setup
+
+* `Wallet-B` is kept offline and turned off most of the time.
+  This greatly reduces attack surface/opportunity.
+* An attacker would have to compromise dcrd on `Computer A` first, and then find a
+  way to jump into `Wallet-B` using the RPC to compromise and exfiltrate data.
+  This is highly unlikely and difficult to pull off.
+* dcrd on `Computer A` can be used for other purposes in your local network (eg. other wallets, DEX, etc),
+  so you don't need to maintain multiple dcrd instances within your local network.
+
+### Setting up Computer A
+
+The best way to setup Decred on your computer is to use [dcrinstall](../wallets/cli/cli-installation.md)
+
+**Note: If you setup dcrwallet in this system, do not use the same seed and passwords for your Wallet-B.**
+
+Once this is done it is assumed that you have dcrd binaries and the dcrd config files.
+
+Note down the IP address of `Computer-A` on your local network.
+The IP can be found using `ifconfig`.
+
+Start dcrd on `Computer-A`, ensuring it exposes RPC to the local network.
+
+`dcrd --rpclisten={LOCALIPOFCOMPUTERA}`
+
+Let dcrd fully sync to the latest block.
+
+We should now start setting up a folder that will be copied over to Wallet-B.
+
+`mkdir ~/copytob`
+
+Copy the certificate and config.
+
+```no-highlight
+cp ~/.dcrd/rpc.cert ~/copytob
+cp ~/.dcrd/dcrd.conf ~/copytob
+```
+
+=== "Raspberry Pi"
+
+    [Download](https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-arm64-v{{ cliversion }}.tar.gz) the Decred binaries for arm64 (Raspberry PI) and place them in the same folder:
+
+    `wget -P ~/copytob https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-arm64-v{{ cliversion }}.tar.gz`
+
+=== "General systems"
+
+    [Download](https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-amd64-v{{ cliversion }}.tar.gz) the Decred binaries for linux and place them in the same folder:
+
+    `wget -P ~/copytob https://github.com/decred/decred-binaries/releases/download/v{{ cliversion }}/decred-linux-amd64-v{{ cliversion }}.tar.gz`
+
+Make sure you [verify](verifying-binaries.md) the tar file to ensure it has not
+been tampered with.
+
+Now you can tar the `copytob` folder for transport.
+
+`tar -zcvf ~/copytob.tar.gz -C ~/copytob decred-linux-{arm/amd}64-v{{ cliversion }}.tar.gz dcrd.conf rpc.cert`
```

Comment:
```suggestion
`tar -zcvf ~/copytob.tar.gz -C ~/copytob decred-linux-{arm/amd}64-v{{ cliversion }}.tar.gz`
```

I believe this is wrong because `dcrd.{conf,cert}` are already in the copytob directory